src/codegen/llvm/Builder.zig

```diff
@@ -6349,8 +7038,11 @@ pub fn init(options: Options) InitError!Builder {
         inline for (.{ 1, 8, 16, 29, 32, 64, 80, 128 }) |bits|
             assert(self.intTypeAssumeCapacity(bits) ==
                 @field(Type, std.fmt.comptimePrint("i{d}", .{bits})));
-        inline for (.{0}) |addr_space|
-            assert(self.ptrTypeAssumeCapacity(@enumFromInt(addr_space)) == .ptr);
+        inline for (.{ 0, 4 }) |addr_space_index| {
```

Comment:
do these need to be `inline`? doesn't look like you're depending on the comptime-ness to me.

Reply:
Without the asserts, `inline` would not be needed.  What I really want is to transform the array at comptime and then iterate the transformed array at runtime.